### ISSUE TITLE
Allow to serve multiple applications at once

### ISSUE THREAD

**Issue by FazioNico** [nrwl/nx#1482]
- [X] I am running the latest version
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed
- [X] I'm reporting the issue to the correct repository (not related to Angular, AngularCLI or any dependency)

## Expected Behavior

Not easy to work with backend + frontend together when backend is build as microservice.
And if you use Angular Console you can't run more than 5 project together. 

## Current Behavior

At my company we currently working with backend as microservice architecture managed by Lernajs. Working fine but looking for moving to Nr to be able to have a better code sharing without npm publishing. 
We have multiple backend service (express) and frontend application (angular), we want to run together some of our backend service with one of our frontend application.
I've try many thing to run multiples application front/backend with only once commande and the only solution have find is using `concurrently` to run multiple `ng serve <project-name>` together but when you have more than 10 project to run, it does not work anymore because it uses too much system

It will be nice to have a better solution for very large entreprise applications architecture

**Comment by FrozenPandaz** [comment]
Would you mind creating a repro for me to try? What is the exact error?

**Comment by FazioNico** [comment]
> Would you mind creating a repro for me to try? What is the exact error?

I've create a simple basic project with very simple backend and frontend here [nx-issue1482](https://github.com/FazioNico/nx-issue1482)
I dont have create more than 5 project but the logic is there. 

It will be nice to have a dev commande to serve multiple project from ./apps folder at once.

**Comment by vsavkin** [comment]
@FazioNico the best option would be to define a custom target that serves a collection of microservices together. You can do it as follows in your `angular.json`:

```
{
  "projects": {
      "frontend-1": {
         "architect": {
            "serve-with-services": {
                "builder": "@nrwl/workspace:run-commands",
                "options": {
                  "commands": [
                      {
                        "command": "ng serve microservice-1"
                      },
                      {
                        "command": "ng serve microservice-2"
                      },
                      {
                        "command": "ng serve frontend1"
                      }
                  ],
                  "parallel": true
                }
             }
         }
      }
   }
}
```

**Comment by FazioNico** [comment]
@vsavkin thanks for this solution. It work find 😉

**Comment by numerical25** [comment]
@vsavkin @FazioNico  There is more missing to this solution. I am using angular 8 and node 12. nx is calling for there to be a `root` property` in the frontend-1 project. This solution could be out dated.

**Comment by stefan-karlsson** [comment]
@numerical25 I just tried out the proposed solution and it works fine in my Angular 8/Nest project. I just executed `ng run <application-name>:serve-with-services`

**Comment by adamm-monek** [comment]
Just coming to this as someone new, couldn't you just do:
`nx run-many --target=serve --all` ?

**Comment by kdawg1406** [comment]
Hi @vsavkin,

I'm using Nx 9.5.1 with Angular 9.1.11

I've tried the above and can't get this to work.  In some of the error messages, I was told the angular.json was out of sync with nx.json so I added the frontend-1 to this file.  Then I started getting different errors.

Can you please describe a solution that works with Nx 9.5.1?

I'm working through the recent Nx Workshop in Pluralsight that has a store and api application. 

Thank you very much!

Karl
